packages/fuels-programs/src/call_utils.rs

```diff
@@ -405,6 +435,18 @@ fn extract_unique_contract_ids(calls: &[ContractCall]) -> HashSet<ContractId> {
         .collect()
 }
 
+pub fn is_missing_output_variables(receipts: &[Receipt]) -> bool {
+    receipts.iter().any(
+        |r| matches!(r, Receipt::Revert { ra, .. } if *ra == FAILED_TRANSFER_TO_ADDRESS_SIGNAL),
+    )
+}
+
+pub fn find_contract_not_in_inputs(receipts: &[Receipt]) -> Option<&Receipt> {
+    receipts.iter().find(
+        |r| matches!(r, Receipt::Panic { reason, .. } if *reason.reason() == PanicReason::ContractNotInInputs ),
+    )
+}
```

Comment:
It seems all callers of this function proceed to do the same thing -- extract and map the contract_id. 

We can make the helper a bit more helpful: 
```rust
pub fn find_id_of_missing_contract(receipts: &[Receipt]) -> Option<Bech32ContractId> {
    receipts.iter().find_map(|receipt| match receipt {
        Receipt::Panic {
            reason,
            contract_id,
            ..
        } if *reason.reason() == PanicReason::ContractNotInInputs => {
            let contract_id = contract_id
                .expect("panic caused by a contract not in inputs must have a contract id");
            Some(Bech32ContractId::from(contract_id))
        }
        _ => None,
    })
}
```

This also avoids the double check whether we're dealing with a Panic receipt (check hidden in the `contract_id` getter).